imap_processing/ultra/l1a/ultra_l1a.py

```diff
@@ -0,0 +1,113 @@
+"""Contains code to perform ULTRA L1a science processing."""
+import dataclasses
+import logging
+from pathlib import Path
+
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import calc_start_time, write_cdf
+from imap_processing.ultra import ultra_cdf_attrs
+from imap_processing.ultra.l0.decom_ultra import ULTRA_AUX, decom_ultra_apids
+
+logger = logging.getLogger(__name__)
+
+
+def xarray_aux(decom_ultra_aux: dict):
+    """Create xarray for auxiliary packet.
+
+    Parameters
+    ----------
+    decom_ultra_aux : dict
+        Parsed data.
+
+    Returns
+    -------
+    dataset : xr.Dataset
+        Data in xarray format.
+    """
+    # Converted time
+    time_converted = []
+    for time in decom_ultra_aux["SHCOARSE"]:
+        time_converted.append(calc_start_time(time))
+
+    epoch_time = xr.DataArray(
+        time_converted,
+        name="Epoch",
+        dims=["Epoch"],
+        attrs=ConstantCoordinates.EPOCH,
+    )
+
+    dataset = xr.Dataset(
+        coords={"Epoch": epoch_time},
+        attrs=ultra_cdf_attrs.ultra_l1a_attrs.output(),
+    )
+
+    for key, value in decom_ultra_aux.items():
+        if key in [
+            "VERSION",
+            "TYPE",
+            "SEC_HDR_FLG",
+            "PKT_APID",
+            "SEQ_FLGS",
+            "SRC_SEQ_CTR",
+            "PKT_LEN",
+        ]:
+            attrs = dataclasses.replace(
+                ultra_cdf_attrs.ultra_support_attrs,
+                catdesc=key,  # TODO: short and long descriptions
+                fieldname=key,
+                label_axis=key,
```

Comment:
I don't think we need axis label for support data, especially for CCSDS header stuff.